### ISSUE TITLE
Add Customizable Header/Footer/'Classification Banners' to Rancher - Backend

### ISSUE THREAD

**Issue by maggieliu** [rancher/rancher#26562]
Original ticket: https://github.com/rancher/rancher/issues/25694

**Comment by kinarashah** [comment]
removed to/test label, needs to be backported to release/v2.4

**Comment by kinarashah** [comment]
Available to test in `2.4.3-rc1`

**Comment by jiaqiluo** [comment]
The enhancement is validated on v2.4.3-rc1 with change UI index to `https://releases.rancher.com/ui/latest-2.4/index.html` 

Now we can set the header and banner and showing either of them or both 
 
```
{
  "banner": {
    "textColor": "#313131",
    "background": "#27AE5F",
    "text": "Testing banner and header on v2.4.3-rc1, with UI index latest-2.4"
  },
  "showHeader": "true",
  "showFooter": "true"
}
```
Result:

![Screenshot_2020-04-21 Rancher](https://user-images.githubusercontent.com/6218999/79927734-87652380-83f5-11ea-8cc4-7adefcbee8d1.png)
